data/data/agent/systemd/units/agent-interactive-console.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Get interactive user configuration at boot
+After=network-pre.target NetworkManager.service pre-network-manager-config.service selinux.service
+Before=network.target network.service agent.service NetworkManager-wait-online.service
+
+[Service]
+Type=oneshot
+TTYPath=/dev/tty15
+ExecStartPre=/usr/bin/chvt 15
+ExecStart="/usr/local/bin/agent-interactive-console.sh"
+ExecStartPost=/usr/bin/chvt 1
+TimeoutStartSec=0
+StandardInput=tty
+TTYVHangup=yes
+TTYVTDisallocate=yes
+
+[Install]
+WantedBy=default.target
+RequiredBy=sshd.service systemd-logind.service getty@tty1.service
```

Comment:
Since the service will require the presence of the agent-tui executable (and we already know that there would another service responsible for fetching it from the ISO), what about adding a [systemd Path](https://www.freedesktop.org/software/systemd/man/systemd.path.html) for delaying the activation of the service until the file is available in a given local, ie `/etc/assisted/agent-tui`?

Reply:
I like it and will add the ConditionPathExists.

Reply:
Done. See [Unit] section.